### PR TITLE
(#1701) Cleanup Footer HTML

### LINKED ISSUE #1701: Cleanup Footer HTML
# Prerequisite
 - [x] Merge #1675 

# Acceptance Criteria 
In Tickets #1584 and #1673 we changed how back to top works. This is a further cleanup task as time ran out in this ticket, and also would have increased juggling complexity. We need to:
- [x] Update example site to remove back to top html.
- [x] Update the documentation site footer to work closer to ncids-js implementation of back to top. (e.g., no flicker on load)
- [x] Update documentation somewhere on how to do back to top if you did not have JS - location TBD
- [x] Create test case and scenario for footer without form. The twig will probably need an update.

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #1701

## Pull Request Details

_Add description_

Closes #<TICKET-NUMBER>

## Author PR Checklist
Items that the author of the PR is responsible for checking before submitted the PR.

### General:
- [x] I have reviewed the acceptance criteria defined in the ticket and ensured the work has been completed.
- [x] The commit message passes [all quality commit message standards](https://github.com/NCIOCPL/ncids/wiki/Creating-Quality-Commit-Messages).
- [x] Unit tests have been updated or created to reflect any javascript changes.
- [x] Storybook scenarios have been updated or created to reflect any html/css/js changes.

### Accessibility:
- [x] WCAG 2.1 Level AA requirements have been met.

### Development:
- [x] Any new or updated javascript code has 100% unit test coverage.
- [x] New or updated breakpoints have regression images.
- [x] Breaking changes have been thoroughly documented in the PR.

## Product Reviewer PR Checklist
Items the product team is responsible for reviewing.

### General:
- [x] There are no unexpected or unapproved regression image changes. 
- [x] Functionality of interactive elements meet the acceptance criteria.
- [x] The product is visually and functionally the same across the different browsers.

### Accessibility:
- [x] AxeDev Tools: there are no new or outstanding accessibility issues introduced in this PR.
- [x] Lighthouse: scores have not noticeably decreased during this PR.
- [x] Wave: there are no new errors or contrast errors introduced in this PR.

## Design Reviewer PR Checklist
Items the design team is responsible for reviewing.  

### General:
- [ ] New or updated features introduced in this PR are developed mobile-first.
- [ ] Breakpoint changes and regression images match those breakpoints.
- [ ] This PR has been tested in all supported browsers at all breakpoints.

## Developer Reviewer PR Checklist
Items the development team is responsible for reviewing.

### General:
- [ ] New code passes code quality standards set by industry standards.
- [ ] The expected Storybook stories have been added or updated for the new or updated feature.
- [ ] The expected unit tests have been added or updated for the new or updated feature.

### Accessibility:
- [ ] VoiceOver: Described content matches with what was expected.
- [ ] Keyboard navigation: new or updated features and content are navigable via the keyboard.

### CODE REVIEW

**Review by andyvanavery31 — APPROVED** [review]
Passes Product review. For awareness, @adrianacastaneda the addition of the Storybook story with no form was added in this ticket, and we will address the Doc Site page with tickets #1335 and #1534

File: docs/content/components/footer.mdx [hunk 1]
```diff
@@ -215,6 +215,18 @@ code_snippets:
             }
           ]
         }
+    - title: Back to Top without JavaScript
+      description: |
+        For sites without JavaScript, you will need to add your own 'back to top' feature to comply with accessibility regulations.
+
+        First create a simple anchor tag at the top of your page. This should be placed at the beginning of where the readable content begins.
+
+        `<a name="top"></a>`
```
Comment (olitharp-nci) [inline docs/content/components/footer.mdx:224]: This is bad guidance.

Not only do we not practice this anywhere on our production builds, specs allow anchors with #top to automatically point to the beginning of the document. 

* https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#sect23
* https://html.spec.whatwg.org/multipage/browsing-the-web.html#scroll-to-the-fragment-identifier
* https://drafts.csswg.org/cssom-view/#scroll-to-the-beginning-of-the-document

File: docs/content/components/footer.mdx [hunk 2]
```diff
@@ -215,6 +215,18 @@ code_snippets:
...
+
+        Next you will need to place an anchor link at the end of your content which references the anchor link above by making the href match the name.
+
+        `<a href='#top' aria-label=”Back To Top”>Back  To Top</a>`
```
Comment (olitharp-nci) [inline docs/content/components/footer.mdx:228]:
```suggestion
        `<a href="#top" aria-label="Back To Top">Back  To Top</a>`
```

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Review by andyvanavery31 — APPROVED** [review]
Requested changes are approved and this passes product review.

**Review by olitharp-nci — APPROVED** [review]

**Pull request merged**